WebUI/static/js/tableDisplay.js

```diff
@@ -87,10 +97,68 @@ function buildTable() {
               <td>${myList[i][1]}</td>
               <td>${myList[i][2]}</td>
               <td>${myList[i][3]}</td>
-              <td>${myList[i][4]}</td>
-              <td>${myList[i][5]}</td>
               `
+    if(targetURL == '/replicatesLatest100'){
+      row =  row +  `
+                    <td>${myList[i][4]}</td>
+                    <td>${myList[i][5]}</td>
+                    `
+    }else if(targetURL.includes('/StudyReplicate') || targetURL.includes('/ConfigReplicate')){
+      row = row + `
+                      <td>${myList[i][4]}</td>
+                      `
+    }else if(targetURL.includes('/StudyConfig')){
+      var row = `<tr>
+                  <td><button onclick="pageRedirection('/ConfigReplicate/'+'${myList[i][3]}');" id="ReplicateBtn">${myList[i][0]}</button></td>
+                  <td><button onclick="pageRedirection('/ConfigReplicate/'+'${myList[i][3]}');" id="ReplicateBtn">${myList[i][1]}</button></td>
+                  <td><button onclick="pageRedirection('/ConfigReplicate/'+'${myList[i][3]}');" id="ReplicateBtn">${myList[i][2]}</button></td>`
+    }else if(targetURL.includes('/study')){
+      var studyID;
+      if(myList[i][1]){
+        studyID = myList[i][1]
+      }else{
+        studyID = 'None'
+      }
+      console.log(studyID)
+      var row = `
+              <tr>
+              <td><button onclick="pageRedirection('/StudyConfig/'+'${studyID}');" id="ConfigBtn">${myList[i][0]}</button></td> 
+              <td><button onclick="pageRedirection('/StudyConfig/'+'${studyID}');" id="ConfigBtn">${myList[i][1]}</button></td> 
+              <td><button onclick="pageRedirection('/StudyConfig/'+'${studyID}');" id="ConfigBtn">${myList[i][2]}</button></td>
+              <td><button onclick="pageRedirection('/StudyReplicate/'+'${studyID}');" id="ReplicateBtn">${myList[i][3]}</button></td>
+              `
+      if(parseInt(myList[i][2])==0 &&  parseInt(myList[i][3])==0 && myList[i][1]){
```

Comment:
Be sure to comment on what you expect those indices to contain - important for the next person to maintain this code, or yourself in a couple months.